isaaclab_arena/assets/object_utils.py

```diff
@@ -59,3 +61,38 @@ def detect_object_type(usd_path: str | None = None, stage: Usd.Stage | None = No
         return ObjectType.ARTICULATION
     else:
         raise ValueError("This should not happen. There is an unknown USD type in the tree.")
+
+
+# Predefined rigid body property configurations for factory assembly tasks
+# High iteration count for precision tasks (peg/hole insertion)
+RIGID_BODY_PROPS_HIGH_PRECISION = sim_utils.RigidBodyPropertiesCfg(
+    disable_gravity=False,
+    max_depenetration_velocity=5.0,
+    linear_damping=0.0,
+    angular_damping=0.0,
+    max_linear_velocity=1000.0,
+    max_angular_velocity=3666.0,
+    enable_gyroscopic_forces=True,
+    solver_position_iteration_count=192,
+    solver_velocity_iteration_count=1,
+    max_contact_impulse=1e32,
+)
+
+# Standard iteration count for gear mesh tasks
+RIGID_BODY_PROPS_STANDARD = sim_utils.RigidBodyPropertiesCfg(
+    disable_gravity=False,
+    max_depenetration_velocity=5.0,
+    linear_damping=0.0,
+    angular_damping=0.0,
+    max_linear_velocity=1000.0,
+    max_angular_velocity=3666.0,
+    enable_gyroscopic_forces=True,
+    solver_position_iteration_count=32,
+    solver_velocity_iteration_count=32,
```

Comment:
I find this name a little confusing. I would say that these settings are not standard, because I assume leaving the settings at their default would be "standard". What do you think about `RIGID_BODY_PROPS_MEDIUM_PRECISION`?